.github/workflows/autoformat.yml

```diff
@@ -158,13 +158,23 @@ jobs:
     if: ${{ needs.format.outputs.reformatted == 'true' }}
     permissions: {}
     steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          # See https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/managing-private-keys-for-github-apps
+          # for how to rotate the private key
+          private-key: ${{ secrets.APP_PRIVATE_KEY }}
```

Comment:
An attacker can till puppet the app if `APP_ID` and `APP_PRIVATE_KEY` are leaked.